### PR TITLE
added note about digitally sighned PDF

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#738

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/set-up-irm-in-sp-admin-center.md [hunk 1]
```diff
@@ -61,7 +61,10 @@ SharePoint Online supports encryption of the following file types:
 - The Office Open XML formats for the following Microsoft Office programs: Word, Excel, and PowerPoint
     
 - The XML Paper Specification (XPS) format
-    
+ 
+> [!NOTE]
+> IRM protection cannot be applied to the protected documents (like digitally signed PDF files) as SharePoint needs to open the document on upload.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/set-up-irm-in-sp-admin-center.md]:
```suggestion
> IRM protection cannot be applied to protected documents (like digitally signed PDF files) as SharePoint needs to open the document on upload. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edit.

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #738

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Any updates on this?

Thank you.

**Comment by KCCross** [comment]
We've just onboarded a new PM in this space and I'm working on getting confirmation about this. The underlying issue has to do with encryption support for PDF in a similar technology that uses IRM. Apologies for the hold-up. This IS on my radar.

**Comment by KCCross** [comment]
Kaarin, requesting assistance on finalizing information for SharePoint and PDF

**Pull request merged**